### PR TITLE
Improve sensitivity example

### PR DESCRIPTION
**Pull request by maxnoe**
**Description**

This updates  the sensitivity example a bit, three main changes:
* The configuration options are changed to match the given IRFs and the standard CTA requirements
* A note that highlights that the PSF is evaluated in true energy to get a containment radius that is then used in reconstructed energy
* Simplify the plotting code

**Dear reviewer**

### CODE REVIEW

**Comment by maxnoe** [comment]
I don't know why the code-meta bot seems to have replaced @adonath with @aaguasca

File: examples/tutorials/analysis-1d/cta_sensitivity.py [hunk 1]
```diff
@@ -83,30 +95,34 @@
...
 containment = 0.68
 
 # correct exposure
 dataset.exposure *= containment
 
 # correct background estimation
+# Warning: this neglects the energy dispersion by computing the containment
+# radius from the PSF in true energy but using the reco energy axis.
 on_radii = obs.psf.containment_radius(
     energy_true=energy_axis.center, offset=0.5 * u.deg, fraction=containment
```
Comment (bkhelifi) [inline examples/tutorials/analysis-1d/cta_sensitivity.py:123]:
```suggestion
    energy_true=energy_axis.center, offset=offset, fraction=containment
```

File: examples/tutorials/analysis-1d/cta_sensitivity.py [hunk 2]
```diff
@@ -83,30 +95,34 @@
...
 spectrum_maker = SpectrumDatasetMaker(selection=["exposure", "edisp", "background"])
 dataset = spectrum_maker.run(empty_dataset, obs)
 
-
 ######################################################################
-# Now we correct for the energy dependent region size:
+# Now we correct for the energy dependent region size.
 #
+# **Note**: In the calculation of the containment radius, we use the point spread function
+# which is defined dependent on true energy to compute the correction we apply in reconstructed
```
Comment (bkhelifi) [inline examples/tutorials/analysis-1d/cta_sensitivity.py:111]: Which correction are you speaking about? on the bkg counts that are defined indeed in the reco energy space...

File: examples/tutorials/analysis-1d/cta_sensitivity.py [hunk 3]
```diff
@@ -83,30 +95,34 @@
...
+# energy, thus neglecting the energy dispersion in this step.
 
 containment = 0.68
 
 # correct exposure
 dataset.exposure *= containment
 
 # correct background estimation
+# Warning: this neglects the energy dispersion by computing the containment
+# radius from the PSF in true energy but using the reco energy axis.
```
Comment (bkhelifi) [inline examples/tutorials/analysis-1d/cta_sensitivity.py:121]: Why are you speaking about reco energy? the containment_radius of the PSF is in true energy, is not it?

**Review by registerrier — APPROVED** [review]
Thanks @maxnoe . 

Technically, the tutorial approach is inconsistent with the IRF it uses. It should use proper pointlike IRFs rather than full containment ones. It might be better emphasized in the introduction. We are trying here to approximate the pointlike IRF from the full containment one.

When instantiating the `SpectrumDatasetMaker` you might also explicitly add the `containment_correction=False` and explain that we have to manually apply it. 

I think you could even split the cell where the correction is applied in two. 
One applying the exposure correction, the other correcting the background.  And explain for the first one explain, that we try to reproduce an IRF where the radmax contains 68% of the signal at any reco energy.

**Comment by maxnoe** [comment]
@registerrier Good idea, will do

File: examples/tutorials/analysis-1d/cta_sensitivity.py (hunk 3 above)
Comment (maxnoe) [reply examples/tutorials/analysis-1d/cta_sensitivity.py:121]: Yes, that's the issue. This uses the PSF to compute a containment radius in true energy that is then applied to reco energy bins of the background to "simulate" the effect of rad max.

The example is computing something in true energy and then applies it in reco energy.

**Review by bkhelifi — DISMISSED** [review]
Thanks @maxnoe

**Comment by maxnoe** [comment]
The comment by @registerrier is not implemented yet

**Review by registerrier — DISMISSED** [review]
Thanks @maxnoe .

**Comment by registerrier** [comment]
@Astro-Kirsty could you try to implement the suggest above about the SpectrumDatasetMaker?

**Comment by Astro-Kirsty** [comment]
Fixed the comments made by @registerrier, the CI fail is unrelated

**Review by registerrier — APPROVED** [review]
Thanks @maxnoe and @Astro-Kirsty . No further comment.

**Pull request merged**